rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -76,6 +76,14 @@ def setup(self) -> None:
         assert self.config.vf_config, "vf_config must be provided."
         assert self.config.encoder_config, "shared encoder config must be " "provided."
 
+        self.config.encoder_config.input_dim = self.config.observation_space.shape[0]
+        self.config.pi_config.input_dim = self.config.encoder_config.output_dim
+        if isinstance(self.config.action_space, gym.spaces.Discrete):
+            self.config.pi_config.output_dim = self.config.action_space.n
+        else:
+            self.config.pi_config.output_dim = self.config.action_space.shape[0] * 2
+        self.config.vf_config.output_dim = 1
+
```

Comment:
This makes sure that not only configs derived from model_config have the correct dims.
One thing that is wrong here still though is that you need output dim to construct these and we set it "silently" behind the scenes. I'm adding this as a TODO in my other PR.